### ISSUE TITLE
PySide6 type stubs are broken

### ISSUE THREAD

**Issue by KevOrr** [Vector35/binaryninja-api#2641]
**Binary Ninja Version**
2.4.2853~2.4.2983

**Describe the bug**
Starting with 2.4.2853, the type stubs in the `python3/PySide6` directory are un-parseable and therefore cannot be used with a Python typechecker such as Mypy or Pyright.

**To Reproduce**
1. Setup:
```py
$ echo $PYTHONPATH
/home/kevin/.local/share/binaryninja/python:/home/kevin/.local/share/binaryninja/python3
$ ls
test_qt5.py  test_qt6.py
$ cat test_qt5.py 
import sys
from PySide2.QtCore import Qt
reveal_type(sys)
reveal_type(Qt)
$ cat test_qt6.py 
import sys
from PySide6.QtCore import Qt
reveal_type(sys)
reveal_type(Qt)
```
2. Downgrade to 2.4.2850 and run
  <div align="left">
    <img src="https://gist.githubusercontent.com/KevOrr/886d461d8630b8bfd6e92c08b94f5978/raw/169475c6717751bae9d8b78a0a4b1206938247a4/qt5.svg">
  </div>

3. Upgrade to 2.4.2853~2.4.2983 and run
<div align="left">
    <img src="https://gist.githubusercontent.com/KevOrr/886d461d8630b8bfd6e92c08b94f5978/raw/169475c6717751bae9d8b78a0a4b1206938247a4/qt6.svg">
</div>

**Expected behavior**
The PySide6 type stubs that ship with the Qt6 builds should be parseable: note how the PySide2 stub correctly has `...` as the class body but the PySide6 stub has no class body, making it syntactically invalid.

**Version and Platform (required):**
 - Binary Ninja: 2.4.2853 and later (tested up to 2.4.2983)
 - OS: Arch Linux
 - Version: 

**Additional context**
I could not identify what version of pyside-setup causes these broken type stubs. I tried bisecting from where they renamed PySide2 to PySide6 ([251807301](https://code.qt.io/cgit/pyside/pyside-setup.git/commit/?id=251807301)) up to where `dev` was a couple weeks ago ([34611847c](https://code.qt.io/cgit/pyside/pyside-setup.git/commit/?id=34611847c)).

Which version of pyside-setup does Binary Ninja use?

**Comment by psifertex** [comment]
FYI, I'm currently testing 6.1.3 to see if this is resolved there. I'm 🤞 that is all it takes.

**Comment by psifertex** [comment]
Given: https://github.com/microsoft/pylance-release/issues/1008#issuecomment-876097061

it seems this should be exactly what we need to do.

**Comment by KevOrr** [comment]
Ooh, nice find. Looks like [6acc53a4](https://code.qt.io/cgit/pyside/pyside-setup.git/commit/?id=6acc53a4e426fee336a8577581f579d6c173c7bc) fixed it, and that's on the 6.1.2 branch. Turns out they were assuming that a class will have an actual body (thus a `...` should not be printed) if it has any methods at all, even if the only methods it has are the typical dunder methods that are inherited by `QObject` or `object`. The fix was to not count those as contributing towards a non-existent class body

**Comment by negasora** [comment]
This was fixed in 2.4.3031-dev
